### PR TITLE
Add vector tests

### PR DESCRIPTION
**Pull request by alexandrebarbaruiva**
## What was done

Added tests to vector class

## Why was it done?

To maintain code quality

Closes: #3
Closes: #9

## How to test locally

Run the tests

## Checklist

- [X] Added some description to this PR
- [X] Assigned yourself
- [X] Assigned a reviewer (@hasher-hasher for now)
- [X] Assigned a related issue (when possible)
- [X] Did tests related to your changes (at least one of the following)
    - [X] Manually
    - [X] Automated

### CODE REVIEW

File: tinyengine/core.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import pygame
-from pygame.locals import *
```
Comment (hasher-hasher) [inline tinyengine/core.py:2]: There are some features that is imported by locals. Did you checked if the code is not using any of those?

Comment (alexandrebarbaruiva) [reply tinyengine/core.py:2]: yep

Comment (alexandrebarbaruiva) [reply tinyengine/core.py:2]: locals are no longer being used

File: tinyengine/engine.py [hunk 2]
```diff
@@ -1,18 +1,14 @@
+import importlib
+import pkgutil
+
 import pygame
-from pygame.locals import *
```
Comment (hasher-hasher) [inline tinyengine/engine.py:2]: There are some features that is imported by locals. Did you checked if the code is not using any of those?

Comment (alexandrebarbaruiva) [reply tinyengine/engine.py:2]: same

**Review by hasher-hasher — APPROVED** [review]
All good. I noticed that the vector class can be used in a way that was not intended. I'll check it later

**Comment by alexandrebarbaruiva** [comment]
> All good. I noticed that the vector class can be used in a way that was not intended. I'll check it later

Add an issue to the backlog for history and keep it tracked!

**Pull request merged**